### PR TITLE
Create tasmota_irhvac.markdown

### PR DESCRIPTION
**Pull request by hristo-atanasov**
## Proposed change

Add 'tasmota_irhvac.markdown'

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:
https://github.com/home-assistant/home-assistant/pull/31422

- This PR fixes or closes issue:

## Checklist

- [ x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by hristo-atanasov** [comment]
What is wrong with these checks? Can someone help me and translate the errors in human readable sentences?

File: source/_integrations/tasmota_irhvac.markdown [hunk 1]
```diff
@@ -0,0 +1,231 @@
+---
+title: Tasmota Irhvac
+description: Control Your Air Conditioner via Home Assistant and Tasmota IR Transceiver
+logo: home-assistant.png
+ha_category:
+  - Climate
+ha_release: pre 0.105.0
```
Comment (klaasnicolaas) [inline source/_integrations/tasmota_irhvac.markdown]:
```suggestion
ha_release: 0.106
```

File: source/_integrations/tasmota_irhvac.markdown [hunk 2]
```diff
@@ -0,0 +1,231 @@
+---
+title: Tasmota Irhvac
+description: Control Your Air Conditioner via Home Assistant and Tasmota IR Transceiver
+logo: home-assistant.png
```
Comment (klaasnicolaas) [inline source/_integrations/tasmota_irhvac.markdown]: Logo from Tasmota?

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_integrations/tasmota_irhvac.markdown (hunk 2 above)
Comment (hristo-atanasov) [reply source/_integrations/tasmota_irhvac.markdown]: I'll have to ask for permissions to use it, first .. :)

Comment (frenck) [reply source/_integrations/tasmota_irhvac.markdown]: At least remove the current logo, it doesn't belong to this integration.
Thanks 👍

Comment (hristo-atanasov) [reply source/_integrations/tasmota_irhvac.markdown]: @klaasnicolaas Can you tell me more on this  Jekyll build (pull_request) test, so I can fix the documentation and pass the test ?

Comment (klaasnicolaas) [reply source/_integrations/tasmota_irhvac.markdown]: @frenck will take a look at your PR tomorrow.

Comment (hristo-atanasov) [reply source/_integrations/tasmota_irhvac.markdown]: Theo from Tasmota gave me a green light for the logo .. So .. I just have to find where to upload it ..

File: source/_integrations/tasmota_irhvac.markdown [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+  required: false
+  default: Off
+  type: string
+default_beep_mode:
+  description: If your AC supports turning `On` or `Off` its own sounds, and you always want to use it, set it to `On`. This option cannot be changed trough HA UI. Recommended value `Off`.
+  required: false
+  default: Off
+  type: string
+default_sleep_mode:
+  description: If your AC supports sleep mode, and you always want to use it, set it to the value of the `Sleep` key in `HVAC`. This option cannot be changed trough HA UI. Recommended value `-1` or the value of `Sleep` key in `HVAC` when sleep mode is turned ff.
```
Comment (klaasnicolaas) [inline source/_integrations/tasmota_irhvac.markdown]: What is **ff**? (last word in the sentence)

File: source/_integrations/tasmota_irhvac.markdown [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+  required: false
+  default: "-1"
+  type: string
+initial_operation_mode:
+  description: The initial operation mode. Used if no retained current operation mode set in HA on starting up, or when turning the AC on trough voice commands via `Google Home` or `Alexa`. Valid values are `heat`, `cool`, `dry`, `fan_only`, `auto` or `off`. Value has to be double quoted.
+  required: false
+  default: "off"
+  type: string
+away_temp:
+  description: Set the temperature used by `preset_mode: away`.
```
Comment (klaasnicolaas) [inline source/_integrations/tasmota_irhvac.markdown]:
```suggestion
  description: "Set the temperature used by `preset_mode: away`."
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_integrations/tasmota_irhvac.markdown (hunk 3 above)
Comment (hristo-atanasov) [reply source/_integrations/tasmota_irhvac.markdown]: It is Off, but without O .. Fixed!

File: source/_integrations/tasmota_irhvac.markdown (hunk 4 above)
Comment (hristo-atanasov) [reply source/_integrations/tasmota_irhvac.markdown]: Shall we double quote each description?

**Pull request closed without merging**